dev-docs/howtoguides/release_a_new_version.md

```diff
@@ -39,174 +49,215 @@ If this is your first time releasing ubuntu-advantage-tools, you'll need to do t
+
+To structure the changelog you can use the other entries as example. But we basically try to keep this order: debian changes, new features/modifications, testing. Within each section, bullet points should be alphabetized.
+
+When reviewing the release PR, please use the following guidelines when reviewing the new changelog entry:
+  * Is the version correctly updated? We must ensure that the new version in the changelog is
+    correct and it also targets the latest Ubuntu release at the moment.
+  * Is the entry useful for the user? The changelog entries should be user focused, meaning
+    that we should only add entries that we think users will care about (i.e. we don't need
+    entries when fixing a test, as this doesn't provide meaningful information to the user).
+  * Is this entry redundant? Sometimes we may have changes that affect separate modules of the
+    code. We should have an entry only for the module that was most affected by it.
+  * Is the changelog entry unique? We need to verify that the changelog entry is not already
+    reflected in an earlier version of the changelog. If it is, we need not only to remove but double
+    check the process we are using to cherry-pick the commits.
+  * Is this entry actually reflected in the code? Sometimes, we can have changelog entries
+    that are not reflected in the code anymore. This can happen during development when we are
+    still unsure about the behaviour of a feature or when we fix a bug that removes the code
+    that was added. We must verify each changelog entry that is added to be sure of their
+    presence in the product.
+
+
+### 2. Set up `review` and `next` branches
+
+```bash
+# Make sure you are up to date
+git switch main
+git pull
+
+# Tag the release candidate
+git tag $version-rc # e.g., 42-rc
+git push origin $version-rc
+
+# Create the next branch
+git switch -c next-v$((version+1))
+dch --newversion $((version+1))~1~devel
+git add debian/changelog
+git commit -m "open next development version"
+git push origin next-v$((version+1))
+
+# Create the review branch
+git switch main
+git switch -c review-v$version
+git push origin review-v$version
+```
+
+Now configure two things:
+1. Change GitHub's "default" branch to `next-v$((version+1))`
+    - This should automatically switch any open PRs against `main` to target `next-v$((version+1))`.
+2. Change the daily PPA build recipe to use `next-v$((version+1))`
+
+And finally, create a draft PR from `main` to `review-v$version` on GitHub.
+
+### 3. Release to our staging PPA
+
+1. Clone the repository into a clean directory and `git checkout $version`.
+    * *WARNING* Build the package in a clean environment. The reason is that the package
+    will contain everything that is present in the folder. If you are storing credentials or
```

Comment:
```suggestion
    * *WARNING* Build the package in a clean environment, because the package
    will contain everything that is present in the folder. If you are storing credentials or
```